executor/ddl_test.go

```diff
@@ -181,6 +181,11 @@ func (s *testSuite3) TestCreateView(c *C) {
 	c.Assert(err.Error(), Equals, ddl.ErrViewWrongList.Error())
 	_, err = tk.Exec("create view v1 (c) as select a,b from t1 ")
 	c.Assert(err.Error(), Equals, ddl.ErrViewWrongList.Error())
+	//view with or_replace flag
+	tk.MustExec("drop table if exists v1")
+	tk.MustExec("create view v1 (c,d) as select a,b from t1")
+	_, err = tk.Exec("create or replace view v1 (c,d) as select a,b from t1 ")
+	c.Assert(err, IsNil)
```

Comment:
Why not use `MustExec` directly?

Reply:
I just need to test if `CREATE OR REPLACE` work as design,

Reply:
You can use `tk.MustExec(`"create or replace view v1 (c,d) as select a,b from t1 ")` to replace the two lines.